src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
```

Comment:
```suggestion
  constructor(name) {
```
it should take only name as param
![Знімок екрана 2023-08-10 о 10 39 30](https://github.com/mate-academy/js_herbivores_and_carnivores/assets/106310796/bc5eb7de-dd22-4170-a13a-dec4b0f0997c)